docs/.gitignore

```diff
@@ -39,4 +39,11 @@ jspm_packages
 
 install-state.gz
 out
-_pagefind
\ No newline at end of file
+_pagefind
+
+# Yarn specific files
+.yarn/
+.yarnrc
+.yarnrc.yml
+yarn.lock
```

Comment:
Oh, we need `yarn.lock` committed for everyone (and CI) to fetch the same exact dependencies.

Reply:
Yes, I think all these files should be in git.

Reply:
OK, I'll revert back to how it was before and not add any of these yarn-related items to the .gitignore file.